README.md

```diff
@@ -49,36 +48,79 @@ const client = new Realtime(ABLY_API_KEY);
 const spaces = new Spaces(client);
 ```
 
-Create a new space instance and have members enter that space:
+### Create or retrieve a space
+
+A `Space` instance can be created, or an existing space retrieved using the `get()` method. You can connect to multiple spaces, but only once per `get()` operation. Spaces are identified by their unique, unicode string name.
+
+You can pass an `offlineTimeout` option when creating a space to specify the length of time in milliseconds after a member loses their connection before that member's object is removed from the space. The default is 120,000ms (2 minutes).
+
+The following is an example of creating a space with the default `offlineTimeout`:
+
+```ts
+const space = spaces.get("exampleSpace");
+```
+
+### Enter or leave a space
+
```

Comment:
Can we also please explain that this method abstracts away connection details of the client? Also, with these conceptual blobs of info, it would be great to add some use-case for each method.